_episodes/21-management.md

```diff
@@ -50,6 +50,23 @@ If you cannot find someone else to bear full responsibility for the Host role, c
 involving helpers in advance of the workshop, or exploring institutional support for event logistics like registration, snack delivery, 
 helper coordination, and emergency contact.
 
+### Co-teaching with other Instructors
```

Comment:
```suggestion
### Instructors: Co-teaching Models
```

Reply:
Since @martinosorb  resolved without commiting, I cannot commit the original.  but the text reply seemed positive, so I've added this again.  If you do *not* want this, resolve again, that's okay.  If you are okay either find the "commit suggestion" button or reply without resolving.